erpnext/accounts/doctype/payment_request/test_payment_request.py

```diff
@@ -178,9 +178,9 @@ def test_payment_channels(self):
 		pr.reload()
 
 		self.assertEqual(pr.payment_channel, "Email")
-		self.assertEqual(pr.mute_email, False)
+		self.assertEqual(pr.mute_email, True)
 
-		self.assertIsNone(pr.payment_url)
+		self.assertEqual(pr.payment_url, PAYMENT_URL)
```

Comment:
There is no penalty in _generating_ a payment url, the flow starts only when _accessing_ it. Providing a fallback url, even on a Shopping Cart flow is a feature: a customer agent can provide that link from the backend for if the original payment on the website had failed for some reason.